about.html

```diff
@@ -0,0 +1,137 @@
+                  </nav>
+                  <div class="mobile-menu">
+                    <img id="cross" src="./images/cross.png" alt="ham">
+                    <ul>
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="about.html"> Join</a></li>
+                        <li><a href="about.html"> Sponsor</a></li>
+                        <li><a href="about.html"> Schedules</a></li>
+                        <li><a href="about.html"> PROGRAMS</a></li>
+                    </ul>
+                    </div>
+                  <!--*******Desktop Navigation Menu******-->
+        <nav class="nav-menu-desktop-about">
+            <ul class="list-about">
+                   <li><a class="header-icon-about" href="#"><i class="fa-brands fa-facebook-f"></i></i></a></li>
+                   <li><a class="header-icon-about" href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                   <li>English</li>
+               <li>My page</li>
+               <li>Log out</li>
+               </ul>
+           </nav>
+           <!--*******Desktop Navigation Menu******-->
+           <nav class="nav-menu-dekhtop2-about">
+            <ul class="list2-about">
+             <li class="summit-logo-about"><a href="#"><img class="logolink-about" src="./images/websummitlogo.png" alt="WebLogo"></a></li>
+                 <li class="nav-links-about"><a href="index.html">Home</a></li>
+                 <li class="nav-links-about"><a href="#">Exhibition</a></li>
+                 <li class="nav-links-about"><a href="#">Join</a></li>
+                 <li class="nav-links-about"><a href="#">Sponsor</a></li>
+                 <li  class="nav-links-about"><a href="#">Schedules</a></li>
+                 <li class="nav-links-about"><a class="border-link" href="#program">PROGRAMS</a></li>
+             </ul>
+     </nav>
+    </header>
+  
+    <article>
+        <!--*******Mobile Headline Section******-->
+<div class="headline-about">
+    <h3 class="topheadingh3-about">Hello Tech World!</h3>
+    <div class="adjustment">
+        <h1 class="topheadingh1-about">Web Global Annual Summit 2023</h1>
+    </div>
+    <div class="paradiv-about"><p class="headlinepara-about">What is Web Summit? <br><br>
+            
+        At a time of great uncertainty for many industries and, indeed, the world itself, we gather policymakers,
+         heads of state, and the founders and CEOs of technology companies and fast-growing startups, to ask a simple 
+         question: Where to next?<br><br>
+
+        See you in Lisbon this November.
+    </p></div>
+    <div class="datediv-about">
+        <p class="locationpara-about">
+            Please contact us per Email for further questions about Web Summit 2023!
+        </p>
+        <h3 class="dateheading-about underline">baqar2023@webSummitProject.org</h3>
+        
+      </div>
+</div>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly use a `section` tag here. i saw your argument on the use of the article tag, but here is a clear difference :

 - The `section` and `article` tags are both used to structure HTML content, but they have slightly different meanings and use cases.

- The `article` tag should be used for independent, self-contained content that could be syndicated or reused independently from the rest of the page. It's typically used for blog posts, news articles, forum posts, and other types of content that can stand alone. An article should also have a title, author, publication date, and other metadata.

- The `section` tag, on the other hand, is used to group related content together. It's often used to divide a page into different sections, such as an introduction, a main content area, and a footer. Each section may contain its own heading, but the content within each section is related to the overall topic of the page. in the case of your project, this is the right use case.

In general, if the content is an independent piece that could stand alone, use the `article` tag. If the content is part of a larger whole and is related to other content on the page, use the `section` tag to group it together.

Reply:
https://github.com/baqar-abbas/Conference-site-Capstone-Project-Module1/pull/1#discussion_r1109517344